### PR TITLE
Add territories to mapping files

### PR DESCRIPTION
**Pull request by chinandrew**
### Description
Add  support for American Samoa, Guam, US Virgin Islands, and Mariana Islands. Hard coded the fips and population numbers in because they're in 4 separate excel files that dont contain the fips codes, just the region names. Link is commented in code and also in [this issue](https://github.com/cmu-delphi/covidcast-indicators/issues/507#issuecomment-747914143). Doesn't include any zip data yet, will need to find those if we want them. I'm not sure if anyone reports data for territories in zip format.

Not really sure how we want to approach backfilling for source that report this (e.g. JHU, which reports at the state level and not at the county level, or safegraph, which had a sample file report at the county level). Converting to draft mode after running checks until we're clear on what needs to happen after merging.

### Changelog
Itemize code/test/documentation changes and files added/removed.
- Add territory populations to fips population generation code
- Update mapping files after rerunning geo_data_proc.py
- Fix a safegraph test that had virgin islands and guam data that were previously dropped

### Fixes 
- #507

### CODE REVIEW

File: _delphi_utils_python/data_proc/geomap/geo_data_proc.py [hunk 1]
```diff
@@ -400,7 +400,14 @@ def create_fips_population_table():
     df_pr = df_pr[~df_pr["fips"].isin(census_pop["fips"])]
     census_pop_pr = pd.concat([census_pop, df_pr])
 
-    census_pop_pr.to_csv(join(OUTPUT_DIR, FIPS_POPULATION_OUT_FILENAME), index=False)
+    # Filled from https://www.census.gov/data/tables/2010/dec/2010-island-areas.html
+    territories_pop = pd.DataFrame({
+        "fips": ["60010", "60020", "60030", "60040", "60050", "66010", "78010", "78020", "78030", "69085", "69100", "69110", "69120"],
+        "pop": [23030, 1143, 0, 17, 31329, 159358, 50601, 4170, 51634, 0, 2527, 48220, 3136]
```
Comment (dshemetov) [inline _delphi_utils_python/data_proc/geomap/geo_data_proc.py:406]: The sources for the FIPS county codes to names can be found [here](https://en.wikipedia.org/wiki/List_of_United_States_FIPS_codes_by_county#cite_note-CensusT-15) and cross-referenced [here](https://www.nrcs.usda.gov/wps/portal/nrcs/detail/national/home/?cid=nrcs143_013697) and [here](http://datadictionary.naaccr.org/default.aspx?c=11&Version=21#Virgin%20Islands%20of%20the%20United%20States). The population sources can be found at the Census [here](https://www.census.gov/data/tables/2010/dec/2010-island-areas.html). I double checked the numbers and all look good :+1:

Comment (dshemetov) [reply _delphi_utils_python/data_proc/geomap/geo_data_proc.py:406]: While there are a couple other territories that have close aid relationships with the United States such as Marshall Islands, the Republic of Palau, and Micronesia, our data sources do not appear to report on these at the moment. Adding support for those can be done as needed in a separate PR.

**Review by dshemetov — APPROVED** [review]

**Comment by krivard** [comment]
Pulling in some Reich lab folks to weigh in on deployment

**Comment by krivard** [comment]
Backissues have been requested for JHU, which we can build with the commitlog tool. 

Safegraph has no backfill (it doesn't issue updates on past values) so we can probably just generate an issue per file, and compute the issue date as the file date + 4.

Once we deploy this we can let it run for a week and then do a database query to figure out what other indicators have data from Territories.

**Comment by chinandrew** [comment]
realized i never moved this out draft form, will do that now.

**Comment by krivard** [comment]
(just double-checking that everything still passes; a lot's happened in three weeks)

**Pull request merged**